### ISSUE TITLE
The hreflang value is not ISO based when used with i18n

### ISSUE THREAD

**Issue by hoiheart** [nuxt-community/sitemap-module#131]
### What problem does this feature solve?

The hreflang produced by nuxt-i18n is based on the iso attribute.
But sitemap does not produce hreflang based on the iso attribute.
I want to be provided with a hook function or a method of using iso property base.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/sitemap-module/issues/c99">#c99</a>)</em></sub></div>

**Comment by NicoPennec** [comment]
@hoiheart thanks for your report 🙏 

Can you give me more details about your hreflang?
- current value
- expected value

In addition, can you share an example of nuxt-i18n + sitemap configuration, please?

**Comment by NicoPennec** [comment]
@hoiheart thanks for your report 🙏 

Can you give me more details about your hreflang?
- current value
- expected value

In addition, can you share an example of nuxt-i18n + sitemap configuration, please?

**Comment by hoiheart** [comment]
hello, @NicoPennec 
I share a demo repository for it.
I wonder if there is a way for hreflang of sitemap to refer to the iso property.

repository: https://github.com/hoiheart/nuxt-issue-demo/tree/sitemap

```javascript
/* nuxt.config.js */ 
{
  ...
  modules: [
    'nuxt-i18n',
    '@nuxtjs/sitemap'
  ],
  i18n: {
    seo: true,
    locales: [
      {
        code: 'en',
        iso: 'en-US' // Will be used as catchall locale by default
      },
      {
        code: 'gb',
        iso: 'en-GB'
      },
      {
        code: 'tw',
        iso: 'zh-TW'
      },
      {
        code: 'cn',
        iso: 'zh-CN'
      }
    ],
    defaultLocale: 'en',
    vueI18n: {
      fallbackLocale: 'en'
    }
  },
  sitemap: {
    i18n: {
      defaultLocale: 'en',
    }
  }
  ...
}
```

### i18n seo head result
![1](https://user-images.githubusercontent.com/25652218/83523291-ee114d00-a51c-11ea-94fe-a691e2eee412.png)


### sitemap xml result
![2](https://user-images.githubusercontent.com/25652218/83523305-f4072e00-a51c-11ea-93dc-45244d86eb4c.png)

**Comment by NicoPennec** [comment]
@hoiheart I just prepared a fix (see [PR](https://github.com/nuxt-community/sitemap-module/pull/132)).

The sitemap module cannot read the options from the nuxt-i18n module.
My proposal is to be compliant with the nuxt-i18n options and so add the `locales` values to the `sitemap.i18n` option:

```js
// nuxt.config.js

{
  ...
  i18n: {
    defaultLocale: 'en',
    locales: [
      { code: 'en', iso: 'en-US' },
      { code: 'gb', iso: 'en-GB' }
    ]
  },
  sitemap: {
    i18n: {
      defaultLocale: 'en',
      locales: [
        { code: 'en', iso: 'en-US' },
        { code: 'gb', iso: 'en-GB' }
      ]
    }
  },
  ...
}

**Comment by hoiheart** [comment]
@NicoPennec 
I will use your proposal!
thank you 👍👍👍

**Comment by NicoPennec** [comment]
I just published the release [2.3.1 ](https://github.com/nuxt-community/sitemap-module/blob/dev/CHANGELOG.md#231-2020-06-04)that includes the patch to support the `sitemap.i18n.locales` option.
